src/frontend/components/Chat/SharedChatComponents/ChatMessageMetadatas/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Text } from 'grommet';
+import { DateTime } from 'luxon';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+import { theme } from '../../../../utils/theme/theme';
+import { ChatAvatar } from '../ChatAvatar';
+
+interface ChatMessageMetadatasProps {
+  msgDatetime: DateTime;
+  msgSender: string;
+}
+
+const messages = defineMessages({
+  senderTitle: {
+    defaultMessage: 'The name of the sender of the message',
+    description:
+      "A title describing the div where the sender's name is displayed",
+    id: 'component.MessageMetadatas.senderTitle',
+  },
+  messageTimeTitle: {
+    defaultMessage: 'The time at which the message was sent',
+    description:
+      'A title describing the div where the sending time is displayed',
+    id: 'component.MessageMetadatas.messageTimeTitle',
+  },
+});
+
+const { chatFonts } = theme.chat;
+
+const SenderNameTextStyled = styled(Text)`
+  letter-spacing: ${chatFonts.primary.letterSpacing};
+  line-height: ${chatFonts.primary.lineHeight};
+`;
+
+const SendTimeTextStyled = styled(Text)`
+  letter-spacing: ${chatFonts.tertiary.letterSpacing};
+`;
+export const ChatMessageMetadatas = ({
+  msgDatetime,
+  msgSender,
+}: ChatMessageMetadatasProps) => {
+  const intl = useIntl();
+  return (
+    <Box
+      direction={'row'}
+      align={'center'}
+      height={'24px'}
+      margin={{
+        bottom: '9px',
+      }}
+    >
+      <ChatAvatar />
+      <Box
+        margin={{
+          left: '10px',
+        }}
+      >
+        <SenderNameTextStyled
+          color={chatFonts.primary.color}
+          size={chatFonts.primary.fontSize}
+          weight={chatFonts.primary.fontWeight as 'bold'}
```

Comment:
why not using directly `bold` here ? IMO this is a hint that there is a problem with the `chatFonts` types. `fontWeight` should not be a `string` but a literal with all possible `weight` values.

Reply:
I'm not using directly `bold` here, because I want to use my chatFonts object, the same way I use it throughout the application.
Concerning `fontweight` that should not be a string, i'm not sure to understand, because this properties is not typed and is already a litteral (with the `font` value). Maybe i'm missing something ?

Reply:
using `chatFonts.primary.fontWeight as 'bold'` makes no sense here. If `fontWeight` value is `bold` then use directly `bold` here. If the value is subject to change then type it correctly. The way you use it is error prone. One day `chatFonts.primary.fontWeight` will have an other value and you won't detect it here.

Reply:
After looking athe the `utils/theme/theme.ts` file, your `chatFonts` variable is not typed at all, typescript infers all the properties are string and you will have to use the `as` keyword in a lot of place IMO. You should create an interface or a type to help the use of this object. WDYT ?

Reply:
I agree with you, I created one, in theme.ts, trying to reuse as much as possible types already provided by Grommet.
see commit [011304d5d63d2f32a3b81de9dd70f7f627e9644f](https://github.com/openfun/marsha/pull/1247/commits/011304d5d63d2f32a3b81de9dd70f7f627e9644f)